examples/pytorch/simmim.py

```diff
@@ -0,0 +1,103 @@
+import torch
+from torch import nn
+import torchvision
+
+from lightly.data import LightlyDataset
+from lightly.data.collate import MAECollateFunction # Same collate as MAE
+from lightly.models import utils
+from lightly.models.modules import masked_autoencoder
+
+
+class SimMIM(nn.Module):
+    def __init__(self, vit):
+        super().__init__()
+        
+        decoder_dim = vit.hidden_dim
+        self.mask_ratio = 0.75
+        self.patch_size = vit.patch_size
+        self.sequence_length = vit.seq_length
+        self.mask_token = nn.Parameter(torch.zeros(1, 1, decoder_dim))
+
+        # same backbone as MAE
+        self.backbone = masked_autoencoder.MAEBackbone.from_vit(vit) 
+
+        # the decoder is a simple linear layer
+        self.decoder = nn.Linear(vit.hidden_dim, vit.patch_size ** 2 * 3)
+        
+
+    def forward_encoder(self, images, batch_size, idx_keep=None):
```

Comment:
Should be `idx_mask` instead of `idx_keep`. We should also remove the default `None` value as `mask_at_index` does not accept `None` as index.